docs/LOGS.md

```diff
@@ -0,0 +1,130 @@
+- [Introduction](#introduction)
+  - [Log Types](#log-types)
+  - [Fields](#fields)
+    - [Access Fields](#access-fields)
+    - [Backend Fields](#backend-fields)
+    - [Daemon Fields](#daemon-fields)
+  - [Features](#features)
+
+# Introduction
+
+_Logs_ are the information being printed in the console upon the execution of Couper. There are a few different [Log Types](#log-types) and several different [Fields](#fields) containing useful information given in said _Logs_ to be remembered, but you can adjust the _Logs'_ appearance and verbosity with a variety of [Features](#features) to make sure that the information you seek is available as easily as possible.
```

Comment:
```suggestion
_Logs_ are the information being printed in the console upon the execution of Couper. There are a few different [Log Types](#log-types) and several different [Fields](#fields) containing useful information given in said _Logs_ to be remembered, but you can adjust the _Logs'_ appearance and verbosity with a variety of [Settings](#Settings) to make sure that the information you seek is available as easily as possible.
```
lets call it settings since its related to the configuration settings block.